src/excursion/Excursion.svelte

```diff
@@ -0,0 +1,105 @@
+<script>
+  import { excursion } from "../store";
+  import { Collection } from "sveltefire";
+  const addToArray = (provisionRef) => {
+    provisionRef.add({});
+  };
+
+  const deleteFromArray = (provisionRef, id) => {
+    provisionRef
+      .doc(id)
+      .delete()
+      .then((r) => console.log(r));
```

Comment:
we should get a better logging solution for development

Reply:
Any propositions?